### PR TITLE
Add 10.7 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

Add the 10.7 release notes

#### Related issue

N/A

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+## 🥳 Mondoo 10.7 is out! This release includes vendor advisory links, improved CLI scanning, and more!
+
+Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)
+
+---
+
+## 🧹 IMPROVEMENTS
+
+### Show numeric asset scores on the CLI
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:18]:
```suggestion
### Show numeric asset scores in the CLI
```

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Understand your precise scores on the CLI with new numeric score values in addition to A-F scores
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:20]:
```suggestion
Understand your precise scores in the cnspec CLI with new numeric score values in addition to A-F scores.
```

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+![Numeric scores](/img/releases/2024-03-12-mondoo-10.7-is-out/numeric.png)
+
+### Add specific vendor advisory sources
+
+Jump right to the source with new direct links to vendor advisories in software advisory pages.
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:26]:
```suggestion
Jump right to the source with new direct links to vendor advisories on software advisory pages.
```

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+![Advisory links](/img/releases/2024-03-12-mondoo-10.7-is-out/vendor_links.png)
+
+### Improved AWS Integration troubleshooting
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:30]:
```suggestion
### Improved AWS integration troubleshooting
```

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Failures happen, so let's get to the root cause faster with new troubleshooting options for AWS integrations. The ... menu in the AWS integrations pages now includes options to force an update of the Lambda code powering the integration along with new options for sending diagnostics logs directly to Mondoo.
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:32]:
```suggestion
Failures happen, so let's get to the root cause faster with new troubleshooting options for AWS integrations. The ... menu in the AWS integrations pages now includes new options that:
- Force an update of the Lambda code powering the integration
- Send diagnostics logs directly to Mondoo
```

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+![Diagnostics information](/img/releases/2024-03-12-mondoo-10.7-is-out/aws_diagnostics.png)
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Fix failures to detect vulnerable versions of system-wide Visual Studio Code installs on Windows.
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:38]:
```suggestion
- Fix failures to detect vulnerable versions of system-wide Visual Studio Code installations on Windows.
```
Am I being too formal?

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+- Fix incorrect pluralization on the assets page.
+- Fix incorrect source links for Debian, Chrome, and Firefox vulnerabilities and Advisories.
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:40]:
```suggestion
- Fix incorrect source links for Debian, Chrome, and Firefox vulnerabilities and advisories.
```

File: releases/2024-03-12-mondoo-10.7-is-out.md (hunk 6 above)
Comment (tas50) [reply releases/2024-03-12-mondoo-10.7-is-out.md:44]: I'm not attached to it the other way

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+### Improved AWS integration troubleshooting
+
+Failures happen, so let's get to the root cause faster with new troubleshooting options for AWS integrations. The ... menu in the AWS integrations pages now includes options to force an update of the Lambda code powering the integration along with new options for sending diagnostics logs directly to Mondoo.
+
+![Diagnostics information](/img/releases/2024-03-12-mondoo-10.7-is-out/aws_diagnostics.png)
+
+## Kubernetes scanning performance improvements
+
+Introduce a new mechanism to reduce the number of calls made during asset discovery. This is helpful when scanning e.g. larger Kubernetes clusters, and allows cnquery/cnspec to incrementally scan every asset one by one without having to scan all of them initially. This performance improvement not only cuts drastically the execution time, it also eliminates the need for reading container images twice from the system, cutting down on I/O load.
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:40]:
```suggestion
We introduced a new mechanism to reduce the number of calls made during asset discovery. This is especially helpful when scanning larger Kubernetes clusters. It lets cnquery and cnspec incrementally scan every asset one by one without having to scan all of them initially. This performance improvement not only drastically cuts the execution time, it also eliminates the need for reading container images twice from the system, cutting down on I/O load.
```

File: releases/2024-03-12-mondoo-10.7-is-out.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+- Fix macOS systems displaying a low vulnerability score but no CVEs or advisories.
+- Add missing available package data when scanning for vulnerabilities on the command line.
+- Fix failures scanning systems with the command line `--incognito` flag.
+- Add missing first-found data to the asset software tab.
+- Respect the `--output` flag when running `cnspec vuln`.
+- Improve the disk/memory usage of container image scans on large Kubernetes clusters.
+- Fix duplicate AWS instance scans.
+- Add support for VMware vSphere/ESXi 8.0U2b vulnerability scanning.
+- Don't show the service accounts button when a Kubernetes integration is still pending.
+- Show unknown instead of 0.0 when a CVSS score has not been published.
```
Comment (misterpantz) [inline releases/2024-03-12-mondoo-10.7-is-out.md:59]:
```suggestion
- Show "unknown" instead of "0.0" when a CVSS score has not been published.
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**